### ISSUE TITLE
Change color of text. 

### ISSUE THREAD

**Issue by MacMeDan** [MessageKit/MessageKit#64]
There is no method to change the color of the text for an `incoming Message` vs `outgoing message`

**Comment by SD10** [comment]
@MacMeDan I was hoping they would use:
```Swift
func messageColor(for message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> UIColor
```
Here you get the message passed in so you can check if the message is from the current sender and return a different UIColor accordingly.

I thought about splitting this into two methods for `incoming` and `outgoing`, but I felt like this method was more flexible because you could have an unlimited number of different colors 🤔

**Comment by fbartho** [comment]
@SD10 I like that API -- because my intended use-case is going to have more types of messages in the stream, and incoming/outgoing is only one of the Axis. I also would tweak the coloring for system messages, location updates, and other events in the message stream.

**Comment by MacMeDan** [comment]
Yes I agree @fbartho and I am sorry guys, I definitely worded this in a way that it seemed like I suggesting just `Incoming` and `Outgoing` that was meant to just be an example use case. I am trying to leave it open ended just a method for providing an override point to accomplish this.

**Comment by SD10** [comment]
This is closed by #66
